charts/lab-partner/values.yaml

```diff
@@ -0,0 +1,52 @@
+# Default values for lab-partner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+mongodb:
+  service:
+    name: lab-partner-mongodb
+
+image:
+  name: lab-partner
+  pullPolicy: IfNotPresent
+  
+secrets:
+    slackSigningSecret: ""
+    slackBotUserOauthAccessToken: ""
+    teamId: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name:
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: ClusterIP
+  port: 3000
+
+ingress:
+  enabled: true
+  annotations: {}
+  host: localhost
+  path: /api/messages
```

Comment:
why is the path hardcoded to `/api/messages`?  do we intend to only have a single route for this?